tests/integration_tests/test_ingest_component_towns_fund.py

```diff
@@ -217,6 +203,13 @@ def test_ingest_with_r4_file_success_with_load_re_ingest(
         },
     )
 
+    programme_junction_rows_first_ingest = ProgrammeJunction.query.all()
+    programme_id_first_ingest = programme_junction_rows_first_ingest[0].programme_id
+    submission_id_first_ingest = programme_junction_rows_first_ingest[0].submission_id
+    project_detail_rows_first_ingest = Project.query.all()
+
+    db.session.commit()
```

Comment:
is this commit superfluous?

Reply:
Have to include or else raises ```A transaction is already begun on this Session.```

Reply:
can you add an inline comment to state it's purpose?